### DISCUSSION TITLE
Plextraktsync with crontab

### DISCUSSION THREAD

**Discussion by Rocknrolldoggie** [Taxel/PlexTraktSync#828] [Support]
Hi all. Please could you help me. Plextraktsync working perfectly if run within the Linux terminal yet won’t update Trakt when run via crontab. Crontab logs shows it runs but it doesn’t update. Any suggestions ? 
thank you.

**Comment by glensc** [comment]
Too many to suggest, pointless to dump them out without any details of your setup.
